packages/react-params/src/initValue.ts

```diff
@@ -30,9 +30,7 @@ export default function getInitValue (registry: Registry, def: TypeDef): unknown
       : {};
   }
 
-  const type = [TypeDefInfo.Compact, TypeDefInfo.Option].includes(def.info)
-    ? (def.sub as TypeDef).type
-    : def.type;
+  const type = [TypeDefInfo.Compact, TypeDefInfo.Option].includes(def.info) ? (def.sub as TypeDef).type : def.type;
```

Comment:
```suggestion
  const type = [TypeDefInfo.Compact, TypeDefInfo.Option].includes(def.info)
    ? (def.sub as TypeDef).type
    : def.type;
```

Assuming this snuck in via prettier?

Reply:
yes indeed. let me reverse that